src/dtoannotations/Validator.java

```diff
@@ -0,0 +1,25 @@
+package dtoannotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Validator {
+
+    private final LengthValidationProcess length = new LengthValidationProcess();
```

Comment:
you don't need these objects, if you will change their methods static, it will be solved.

Method name **MUST** start in **lowercase**